### DISCUSSION TITLE
how using projact Wpf to Uno

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#4997] [Q&A] [answered]
how using projact Wpf to Uno

**Comment by AbdalaMask** [comment]
How view Project view WPF to WebAssembly and iOS, macOS and Android

**Reply by jeromelaban** [reply]
Thanks for your question. Unfortunately it doest provide enough details on your problem, can you provide more details ?

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/105495030-5091a600-5cc4-11eb-9927-474a9864113f.png)
https://github.com/AbdalaMask/EVStudio.Scada.Platforms

**Reply by AbdalaMask** [reply]
controls HMI not Show to Uwp and web and Android

**Reply by AbdalaMask** [reply]
How to create and use custom control work to All

**Reply by AbdalaMask** [reply]
see
![image](https://user-images.githubusercontent.com/37013638/105497657-d2cf9980-5cc7-11eb-8374-0f0bac6b6e07.png)

**Reply by jeromelaban** [reply]
Thanks. What happens in the other platforms ? Can you share a screenshot of WebAssembly, as well as the logs from the browser console ?

**Reply by AbdalaMask** [reply]
ok

**Reply by AbdalaMask** [reply]
Other tools are not shown
![image](https://user-images.githubusercontent.com/37013638/105500323-6d7da780-5ccb-11eb-985d-31fe1dfbb0c1.png)

**Reply by AbdalaMask** [reply]
Other tools do not show in web and Android

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/105500688-e4b33b80-5ccb-11eb-8279-630bb340c353.png)
and Uwp

**Comment by AbdalaMask** [comment]
To see the project
to link
https://github.com/AbdalaMask/EVStudio.Scada.Platforms

**Comment by AbdalaMask** [comment]
try projact Uwp only 
![image](https://user-images.githubusercontent.com/37013638/105504330-4d041c00-5cd0-11eb-9130-35a650cf64d7.png)
good

**Comment by AbdalaMask** [comment]
hi

**Comment by francoistanguay** [comment, marked as answer]
Not sure what you're trying to accomplish Uno will only bring UWP or WinUI projects to other paltforms. If you want to reuse WPF code or an entire app, you will need to first port it to UWP/WinUI and then use Uno to bring it to WebAssembly or other platforms.
